dynalab_cli/utils.py

```diff
@@ -146,11 +147,11 @@ def default_filename(key):
 
 
 def check_model_name(name):
-    pat = re.compile("^[a-zA-Z0-9-]+$")
+    pat = re.compile("^[a-z0-9-]+$")
```

Comment:
Sagemaker jobs uses the model name and need to be lowercased.

Reply:
I think the model name will be lower cased when creating a job.

Reply:
I don't think sagemaker has that requirements. We've got a couple of job names with upper case